BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
```

Comment:
There are some edge cases here, e.g. multiple current affiliations.

Reply:
There is affiliation at the time the data was generated, which may be different from what is logged at ORCID, which may be different from current affilitation. Do we have any wish to capture affiliation at the time the data was generated as well as current?

Reply:
on the dandi side affiliation would be stored at the point of publishing.